.gitignore

```diff
@@ -26,3 +26,4 @@ _site
 /docs/.jekyll-metadata
 coverage.txt
 cspell.json
+*~
```

Comment:
Why this change is needed?

Reply:
This is to exclude emacs backup files. I use this text editor for development.